handlers/dialog360/handler.go

```diff
@@ -549,7 +533,7 @@ func (h *handler) SendLegacy(ctx context.Context, msg courier.MsgOut, clog *cour
 
 					payload.Interactive = &interactive
 				} else {
-					return nil, fmt.Errorf("too many quick replies WAC supports only up to 10 quick replies")
+					return fmt.Errorf("too many quick replies WAC supports only up to 10 quick replies")
```

Comment:
should just truncate to 10 and log channel log error like we've done in other WhatsApp channels

Reply:
Sure, I will add that here as well